### PR TITLE
Add RavenPack keynote information

### PR DESCRIPTION
**Pull request by easye**
Text and pictures of the precis for the RavenPack keynote present with this commit.

Not functional as I understand it as the program for the first day has to fleshed out for the templating system to kick off.

### CODE REVIEW

File: editions/2021.lisp [hunk 1]
```diff
@@ -185,6 +185,19 @@
...
 
+
+(define-person (:family-name "Lawson"
+                :given-name "Andrew")
+  :role (:speaker)
+  :organization "RavenPack"
+  :address (:country "Spain"))
+
+(define-person (:family-name "Grinis"
+                :given-name "Inna1")
```
Comment (marcoheisig) [inline editions/2021.lisp:196]: This should be "Inna", not "Inna1", right?

Comment (marcoheisig) [reply editions/2021.lisp:196]:
```suggestion
                :given-name "Inna")
```

**Review by marcoheisig — COMMENTED** [review]
Also, I assume the file 'andrew-lawson.jog' should be called 'andrew-lawson.jpg'.

Apart from that, it looks good.  Thanks easye, and thanks RavenPack!

**Comment by easye** [comment]
Superseded by <https://github.com/european-lisp-symposium/els-web/pull/24>

**Pull request closed without merging**